spec/controllers/idv/phone_controller_spec.rb

```diff
@@ -78,6 +78,23 @@
       end
     end
 
+    context 'when the user has chosen to use a different number' do
+      let(:step) { 'path_where_user_asked_to_use_different_number' }
```

Comment:
Nit: Not a path anymore.

```suggestion
      let(:step) { 'step_where_user_asked_to_use_different_number' }
```

Reply:
yeah, I caught that, too! 😄